build/pause/Makefile

```diff
@@ -18,7 +18,7 @@ REGISTRY ?= staging-k8s.gcr.io
 IMAGE = $(REGISTRY)/pause
 IMAGE_WITH_ARCH = $(IMAGE)-$(ARCH)
 
-TAG = 3.2
+TAG = 3.3
```

Comment:
we should make sure we follow up on this and let some folks know they need to do this again.

fyi @dims for containerd

Reply:
ACK!

Reply:
Ack @BenTheElder

Reply:
Now both of you get some sleep! it's an order